conftest.py

```diff
@@ -354,10 +354,10 @@ def remove_upgrade_tests_based_on_config(
         cnv_source(str): cnv source option.
         upgrade_tests(list): list of upgrade tests.
     """
```

Comment:
lgtm
(Is it intentionally in 4.18 branch?)

Reply:
+1 to what @jpeimer said. PRs should be started with main, unless unblocking a release. 4.18 is already out. So this PR should be onhold against 4.18 and we should start with main.